server/WebAPI/DTO/GooleOauthConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebAPI.DTO
+{
+  public class GooleOauthConfiguration
+  {
+    public string ClientId { get; set; }
+    public string ClientSecret { get; set; }
```

Comment:
why do we need ClientSecret in DTO?

Reply:
It isn't DTO, it's just Configuration object. It uses to contain data from appsettings.json